src/server/scripts/EasternKingdoms/MagistersTerrace/boss_felblood_kaelthas.cpp

```diff
@@ -228,8 +289,15 @@ struct boss_felblood_kaelthas : public BossAI
     void UpdateAI(uint32 diff) override
     {
         _OOCScheduler.Update(diff);
+        if (!UpdateVictim())
+            return;
+
+        if (!me->HasUnitState(UNIT_STATE_CASTING))
+            DoMeleeAttackIfReady();
```

Comment:
this duplicates DoMeleeAttackIfReady() that is called inside BossAI::UpdateAI() a few lines below

Reply:
those changes are not needed at all

![image](https://github.com/user-attachments/assets/163d33d6-0a4f-4afc-ab22-6019ccb624e8)